hack/update-generated-protobuf-dockerized.sh

```diff
@@ -41,6 +41,43 @@ fi
 
 gotoprotobuf=$(kube::util::find-binary "go-to-protobuf")
 
+PACKAGES=(
```

Comment:
Any way we can auto-detect these?  We have far too many such lists in far too many places.

Reply:
With a tag? That would be the canonical answer, wouldn't it?

```
// +protobuf-gen:true
```